samples/IHFDM/PropertySetWrapper.h

```diff
@@ -2,9 +2,9 @@
 
 #include <IPropertySet.h>
 #include <CppRecorder.hpp>
-#include <propertysets/properties/array_property.h>
-#include <propertysets/properties/reference_property.h>
-#include <propertysets/properties/string_property.h>
+#include "IPropertySet.h"
+//#include <propertysets/properties/reference_property.h>
```

Comment:
I like your solution for the missing property sets headers - I would prefer to delete the old #include lines rather than comment them out. Thanks!!

Reply:
I'll fix it later 👍